src/github/issue-comment.ts

```diff
@@ -98,10 +98,66 @@ const syncIssueCommentsToJira = async (jiraHost: string, context: WebhookContext
 		context.log.info("Halting further execution for syncIssueCommentsToJira as JiraClient is empty for this installation");
 		return;
 	}
+
```

Comment:
Can you add some tests on this bit?

Reply:
Discussed on [slack](https://atlassian.slack.com/archives/C0496DT7MQV/p1693799083300639) and closing comment